app/marketing/management/commands/expiration.py

```diff
@@ -46,4 +46,4 @@ def handle(self, *args, **options):
                         email_list.append(fulfiller.fulfiller_email)
                     elif fulfiller.profile and fulfiller.profile.email:
                         email_list.append(fulfiller.profile.email)
-                bounty_expire_warning(b, email_list)
+                bounty_expire_warning(b, set(email_list))
```

Comment:
With this approach, you'd have to cast the set back to a list. `(b, list(set(email_list)))` -  Additionally, we might want to investigate performing a `distinct` on `email` for these lists before it makes it to this point.

Reply:
Something like this?

```
for b in bounties:
  email_list = {}
   if b.bounty_owner_email:
      email_list.add(b.bounty_owner_email)
      for fulfiller in b.fulfillments.all():
            if fulfiller.fulfiller_email:
                  email_list.add(fulfiller.fulfiller_email)
            elif fulfiller.profile and fulfiller.profile.email:
                  email_list.add(fulfiller.profile.email)
       bounty_expire_warning(b, list(email_list))
```